src/code/audio_playback.c

```diff
@@ -79,9 +79,9 @@ void Audio_InitNoteSub(Note* note, NoteSubEu* sub, NoteSubAttributes* attrs) {
     } else if (gAudioContext.soundMode == 3) {
         sub->bitField0.s.stereoHeadsetEffects = false;
         sub->bitField0.s.usesHeadsetPanEffects = false;
-        volLeft = 0.707f;
+        volLeft = 0.707f; // approx 1/sqrt(2)
```

Comment:
in another place you used "about" instead of "approx", should probably decide on something for consistency

or sometimes there's no "approx" or equivalent at all: `0.2333333f); // 7 / 30`, `0.7071) { // cos(45 degrees)`

so just no "approx"/"about" ?

Reply:
what about
```suggestion
        volLeft = 0.707f; // ~ 1/sqrt(2)
```
or `~=`
or `≈` but I don't think IDO would like that character